app/facades/food_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodFacade
+  def self.searched_food(food_type)
+    foods = FoodService.get_foods_by_type(food_type)[:foods]
+    foods.map do |food_data|
+      Food.new(food_data)
+    end
+  end
```

Comment:
We could use this method to return both data points needed for the user story:
```
def self.searched_food(food_type)
  food_data = FoodService.get_foods_by_type(food_type)
  foods = food_data[:foods].first(10).map do |food|
    Food.new(food)
  end
  [food_data[:totalHits], foods]
end
```
The controller would assign instance vars like so: 
`@result_count, @foods = FoodFacade.searched_food(params[:q])`